File: .ProgramingBasics/SimpleCalculations/08.TriangleArea/src/TriangleArea.java
import java.util.Scanner;

public class TriangleArea {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a = scanner.nextDouble();
        double h = scanner.nextDouble();

        double area = a * h / 2;

        System.out.println("Triangle area = " + area);
    }
}

File: ProgramingBasics/SimpleCalculations/08.TriangleArea/08.TriangleArea/TriangleArea.cs
using System;

namespace _08.TriangleArea
{
    public class TriangleArea
    {
        public static void Main(string[] args)
        {
            var a = double.Parse(Console.ReadLine());
            var h = double.Parse(Console.ReadLine());
            var area = a * h / 2;

            Console.WriteLine($"Triangle area = {area}");
        }
    }
}
